### PR TITLE
Add section about volume increase

### PR DESCRIPTION
**Pull request by erthalion**
Another important feature to document

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -242,3 +242,34 @@ metadata:
 
 Note that timezone required for `timestamp` (offset relative to UTC, see RFC
 3339 section 5.6)
+
+## Increase volume size
+
+PostgreSQL operator supports statefulset volume resize if you're using the
+operator on top of AWS. For that you need to apply manifest with a new size,
```
Comment (alexeyklyukin) [inline docs/user.md]: I'd change it something like (not clear about which size you are talking about, could be more than one volume that needs to be changed):

'you need to change the size field of the volume description in the cluster manifest and apply it; the operator compares the new volume size with the previous on and acts on differences.'

File: docs/user.md [hunk 2]
```diff
@@ -242,3 +242,34 @@ metadata:
...
+kind: postgresql
+
+metadata:
+  name: acid-test-cluster
+spec:
+  volume:
+    size: 5Gi # new volume size
+```
+
+You can only increase a volume size in this way, if a new requested size is
```
Comment (alexeyklyukin) [inline docs/user.md]: You can only enlarge the volume with the process described above, shrinking is not supported and will emit a warning.

File: docs/user.md [hunk 3]
```diff
@@ -242,3 +242,34 @@ metadata:
...
+smaller than the previous one nothing will be done. After this update all the
+new pods in a statefulset will be created with a new volume size. To increase a
+volume size on already existing pods in a statefulset, the operator will
```
Comment (alexeyklyukin) [inline docs/user.md]: to enlarge persistent volumes attached to the running pods, the operator performs the following actions:

File: docs/user.md [hunk 4]
```diff
@@ -242,3 +242,34 @@ metadata:
...
+perform the following steps:
+
+* modify EBS volume size
```
Comment (alexeyklyukin) [inline docs/user.md]: call AWS API to change the volume size (AWS rate-limits this operation to no more than once every 6 hours)

File: docs/user.md [hunk 5]
```diff
@@ -242,3 +242,34 @@ metadata:
...
+
+* resize an actuall filesystem use `resize2fs`
```
Comment (alexeyklyukin) [inline docs/user.md]: resize the filesystem with `resize2fs` by doing the equivalent of `kubectl exec` on the pod.

File: docs/user.md [hunk 6]
```diff
@@ -242,3 +242,34 @@ metadata:
...
+
+Note that if before a volume size was increased a statefulset was scaled down
```
Comment (alexeyklyukin) [inline docs/user.md]: something like '
if the statefulset is scaled down before resizing the size changes are only applied to the volumes attached to the running pods; the size of the volumes that correspond to the previously running pods is not changed.'

**Comment by alexeyklyukin** [comment]
👍

**Comment by erthalion** [comment]
:+1:

**Pull request merged**